packages/@glimmer/runtime/lib/vm/element-builder.ts

```diff
@@ -215,6 +217,29 @@ export class NewElementBuilder implements ElementBuilder {
     guid: string,
     insertBefore: Maybe<SimpleNode>
   ): Nullable<RemoteLiveBlock> {
+    if (this.env.debugRenderTree) {
+      const state = {};
+      this.remoteStateStack.push(state);
+      const ref = createUnboundRef(element, false);
+      this.env.debugRenderTree.create(state, {
+        type: 'remote-element',
+        name: 'in-element',
+        args: {
+          positional: [],
+          named: {
+            destination: ref,
+          },
+        } as any,
+        instance: {
+          args: {
+            destination: ref,
+          },
+          constructor: {
+            name: 'InElement',
+          },
+        },
+      });
```

Comment:
```suggestion
      this.env.debugRenderTree.create(state, {
        type: 'keyword',
        name: 'in-element',
        args: createCapturedArgs(
          // not sure why the order is backwards, but it's named then positional
          { insertBefore: createConstRef(insertBefore, false) },
          [createConstRef(element, false)],
        ),
        instance: null,
      });
```

Reply:
*see the other comment regarding `state`

Reply:
Hm, let's see how we can resolve that, what was the cycle?

Reply:
I'm not entirely sure

I think
* arguments
* stack
* low-level
* opcodes
* element-builder

But i also saw this:

* arguments
* compiled/opcodes/-debug-strip
* ../vm/arguments

Reply:
The error happens in glimmer/runtime/lib/compiled/opcodes/vm.ts:38:1
APPEND_OPCODES is not defined